client/src/components/menubar/index.tsx

```diff
@@ -109,9 +117,18 @@ class MenuBar extends React.PureComponent<MenuBarProps, State> {
 
     this.state = {
       pendingClipPercentiles: INITIAL_PERCENTILES,
+      mediaWidthMatches: window.matchMedia("(max-width: 1275px)").matches,
     };
   }
 
+  componentDidMount() {
+    const handler = (e: MediaQueryListEvent) =>
```

Comment:
maybe we need to make this a method, so we can remove the handler in `componentWillUnmount()`?